subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -26,6 +27,8 @@
 
 LOG_MODULE_REGISTER(nrf_cloud_transport, CONFIG_NRF_CLOUD_LOG_LEVEL);
 
+#define UNUSED_VARIABLE(X) ((void)(X))
```

Comment:
The Zephyr macro `ARG_UNUSED(x)` can also be used for this purpose

Reply:
ah thanks, i thought there was a common one defined somewhere